Modules/Components/Elastix/include/selxMonolithicTransformixComponent.hxx

```diff
@@ -128,16 +128,20 @@ MonolithicTransformixComponent< Dimensionality, TPixel >::Set( typename elastixT
   return 0;
 }
 
-
-//Since elastixFilter is not a true itkfilter we cannot use itkImageSourceInterface (yet)
-
 template< int Dimensionality, class TPixel >
 typename MonolithicTransformixComponent< Dimensionality, TPixel >::ResultImageType::Pointer
 MonolithicTransformixComponent< Dimensionality, TPixel >::GetItkImage()
 {
   return this->m_transformixFilter->GetOutput();
 }
 
+template< int Dimensionality, class TPixel >
+typename MonolithicTransformixComponent< Dimensionality, TPixel >::DisplacementFieldImageType::Pointer
+MonolithicTransformixComponent< Dimensionality, TPixel >::GetDisplacementFieldItkImage()
+{
+  this->m_transformixFilter->ComputeDeformationFieldOn();
+  return this->m_transformixFilter->GetOutputDeformationField();
```

Comment:
Check if DeformationField is actually created if no ResultImage was requested (by itk pipeline update)